[functorch] re-enable some linalg.det tests

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #83619
* #83618
* #83614
* #83612
* #83178
* #83597
* #83530

To avoid regressions. Fixes
https://github.com/pytorch/functorch/issues/961

Test Plan:
- wait for Ci

zou3519 committed:
[functorch] re-enable some linalg.det tests

To avoid regressions. Fixes
https://github.com/pytorch/functorch/issues/961

Test Plan:
- wait for Ci

[ghstack-poisoned]

zou3519 committed:
Update on "[functorch] re-enable some linalg.det tests"


To avoid regressions. Fixes
https://github.com/pytorch/functorch/issues/961

Test Plan:
- wait for Ci

[ghstack-poisoned]

Lezcano commented:
fwiw, the `singular` test was still failing in https://github.com/pytorch/pytorch/pull/80217.

zou3519 committed:
Update on "[functorch] re-enable some linalg.det tests"


To avoid regressions. Fixes
https://github.com/pytorch/functorch/issues/961

Test Plan:
- wait for Ci

[ghstack-poisoned]

zou3519 commented:
out of date
